modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ViewObjectDefinitionsDisplayContext.java

```diff
@@ -133,28 +134,7 @@ public FDSSortItemList getFDSSortItemList() {
 		).build();
 	}
 
-	public PortletURL getPortletURL() throws PortletException {
-		return PortletURLUtil.clone(
-			PortletURLUtil.getCurrent(
-				_objectRequestHelper.getLiferayPortletRequest(),
-				_objectRequestHelper.getLiferayPortletResponse()),
-			_objectRequestHelper.getLiferayPortletResponse());
-	}
-
-	public JSONArray getStoragesJSONArray() throws Exception {
-		return JSONUtil.toJSONArray(
-			_objectEntryManagerRegistry.getObjectEntryManagers(
-				_objectRequestHelper.getCompanyId()),
-			objectEntryManager -> JSONUtil.put(
-				"label",
-				objectEntryManager.getStorageLabel(
-					_objectRequestHelper.getLocale())
-			).put(
-				"type", objectEntryManager.getStorageType()
-			));
-	}
-
-	private String _getPermissionsURL() throws Exception {
+	public String getPermissionsURL(String modelResource) throws Exception {
```

Comment:
Could you rename the variable to "className" or "modelName" or "resourceName"?
I saw usage for those names.

Reply:
okay, we can keep it this way.